reference/conanfile/attributes.rst

```diff
@@ -1257,7 +1257,8 @@ scm
 
 .. warning::
 
-    This is an **experimental** feature subject to breaking changes in future releases.
+    Depending on the use, this is feature may be considered **experimental** and subject to breaking
+    changes in future releases. See :ref:`note <scm_to_conandata_note>` below.
```

Comment:
Same here, I would keep the "experimental warning/note" everything inside a single callout.